syntax = "proto3";

package grpc_merger;

service MergerCommunication {
    rpc pushData (MergerDataRequest) returns (MergerDataReply) {}
    rpc ConnCheck (ConnCheckRequest) returns (ConnCheckResponse) {}
}

message MergerDataRequest{
    bytes data = 1;
}

message MergerDataReply{
    bool checker = 1;
}

message ConnCheckRequest{
    bool checker = 1;
}

message ConnCheckResponse{

}

